lib/src/command/deps.dart

```diff
@@ -126,7 +126,9 @@ class DepsCommand extends PubCommand {
           'version': currentPackage.version.toString(),
           'kind': kind,
           'source': source,
-          'dependencies': next,
+          'dependencies': currentPackage.dependencies.keys.toList(),
+          if (isRoot)
+            'dev_dependencies': currentPackage.devDependencies.keys.toList(),
```

Comment:
Should this be `devDependencies`? I don't know about the rest of this file, but files like `package_config.json` seem to use camel case (for ex. `rootUri`)

Reply:
Good question - in pubspec.yaml we use `dev_dependencies` - but json guidelines generally suggest camelCase.

@jonasfj do you have an opinion?

Reply:
I don't know, I guess using `devDependencies` is the most consistent with other JSON outputs we have.

Reply:
Done